src/hor2eq.jl

```diff
@@ -0,0 +1,143 @@
+# This file is a part of AstroLib.jl. License is MIT "Expat".
+
+function _hor2eq(alt::T, az::T, jd::T, lat::T, lon::T, altitude::T,
+                 pressure::T, temperature::T, ws::Bool, B1950::Bool,
+                 precession::Bool, nutate::Bool, aberration::Bool,
+                 refract::Bool,obsname::AbstractString) where {T<:AbstractFloat}
+
+    if obsname == ""
+        # Using Pine Bluff Observatory values
+        if isnan(lat)
+            lat = T(43.0783)
+        end
+
+        if isnan(lon)
+            lon = T(-89.865)
+        end
+    else
+        lat = observatories[obsname].latitude
```

Comment:
Thanks. I had done that till f818e8e actually, changed that in attempt for type stability and didn't revert it back later